synapse/models/infotech.py

```diff
@@ -1636,6 +1646,25 @@ def getModelDefs(self):
                     ('author:contact', ('ps:contact', {}), {
                         'doc': 'The contact information for the author of the ATT&CK Flow diagram.'}),
                 )),
+                ('it:mitre:attack:datasource', {}, (
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the datasource.'}),
+                    ('description', ('str', {}), {
+                        'doc': 'A description of the datasource.'}),
+                    ('references', ('array', {'type': 'inet:url', 'uniq': True}), {
```

Comment:
should this also be sorted?

Reply:
None of the other `it:mitre:attack:*` forms have this property sorted. I think it would be better to maintain consistency?

Reply:
Yeah probably better to keep it consistent but I'm not sure that it was left off intentionally.

Reply:
Updated - might as well start digging up :)